cli/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cli"
+version = "0.0.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tokio = { workspace = true, features = ["full"] }
+db = { path = "../db" }
+color-eyre = { workspace = true }
+
+[lints]
+workspace = true
```

Comment:
Each crate needs to opt-in to the clippy lints that the workspace setup, kinda annoying but 🤷